packages/nestjs/package.json

```diff
@@ -46,9 +46,9 @@
   "dependencies": {
     "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/core": "^1.30.1",
-    "@opentelemetry/instrumentation": "0.57.1",
-    "@opentelemetry/instrumentation-nestjs-core": "0.44.0",
-    "@opentelemetry/semantic-conventions": "^1.27.0",
+    "@opentelemetry/instrumentation": "0.57.2",
+    "@opentelemetry/instrumentation-nestjs-core": "0.44.1",
```

Comment:
We are actually not using this package in the Nest.js package. I think it's just there so we don't forget about upgrading it until [this PR](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2685) is merged.

For the time being, we have inlined the instrumentation [in this PR](https://github.com/getsentry/sentry-javascript/pull/15114). 

Basically, we only changed this line: `const supportedVersions = ['>=4.0.0 <12'];` 

But I'm not sure if it's worth it to update the inlined instrumentation.

Reply:
Let's keep it upgraded so that all the underlying deps have the same versions.